tests/run-make/wasm-unexpected-features/rmake.rs

```diff
@@ -0,0 +1,25 @@
+//@ only-wasm32v1-none
+
+use std::path::Path;
+
+use run_make_support::{rfs, rustc, wasmparser};
+
+fn main() {
+    rustc()
+        .input("foo.rs")
+        .target("wasm32v1-none")
+        .lto("fat")
+        .linker_plugin_lto("on")
+        .link_arg("--import-memory")
+        .opt_level("z")
+        .run();
+    verify_features(Path::new("foo.wasm"));
+}
+
+fn verify_features(path: &Path) {
+    eprintln!("verify {path:?}");
+    let file = rfs::read(&path);
+
+    let mut validator = wasmparser::Validator::new_with_features(wasmparser::WasmFeatures::WASM1);
```

Comment:
`wasm32v1-none` produces `LIME1` compatible files, not `WASM1` compatible ones. lime1 is a slight superset of wasm1.

Reply:
it's `target-cpu = mvp` and `+mutable-globals`

Reply:
https://github.com/rust-lang/rust/blob/master/compiler/rustc_target/src/spec/targets/wasm32v1_none.rs